File: class02/cpp/AP0211-ContagemRegressivaFlux.cpp
#include <iostream>
int main() {
    int n;
    std::cout << "Digite um número: "; std::cin >> n;
    while (n >= 0) { std::cout << n << std::endl; n--; }
    std::cout << "Fogo!" << std::endl;
    return 0;
}

File: class02/java/AP0211-ContagemRegressivaFlux.java
import java.util.Scanner;
public class ContagemRegressivaFlux {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite um número: ");
        int n = sc.nextInt();
        while (n >= 0) { System.out.println(n); n--; }
        System.out.println("Fogo!");
        sc.close();
    }
}